tests/test_fragment.py

```diff
@@ -0,0 +1,25 @@
+from htpy import Fragment, i, iter_node, p
+
+
+def test_render_direct() -> None:
+    assert str(Fragment("Hello ", None, i["World"])) == "Hello <i>World</i>"
+
+
+def test_render_as_child() -> None:
+    assert (
+        str(p["Say: ", Fragment("Hello ", None, i["World"]), "!"])
+        == "<p>Say: Hello <i>World</i>!</p>"
+    )
+
+
+def test_render_nested() -> None:
+    assert str(Fragment(Fragment("Hel", "lo "), None, i["World"])) == "Hello <i>World</i>"
+
+
+def test_render_chunks() -> None:
+    assert tuple(iter_node(Fragment("Hello ", None, i["World"]))) == (
+        "Hello ",
+        "<i>",
+        "World",
+        "</i>",
+    )
```

Comment:
nice tests!

I think it would be better if they used the render fixture like all other tests. I (think) I plan to eventually add async support (see #38). then the render fixture will make sure that everything works async too.